tests/testthat/assets/badges-a-la-usethis/README.md

```diff
@@ -0,0 +1,23 @@
+# thisisatest <img src='man/figures/logo.png' align="right" height="138.5" />
+
+> Connect to thisisatest, from R
+
+<!-- badges: start -->
+[![Linux Build Status](https://travis-ci.org/thisisatest/thisisatest.svg?branch=master)](https://travis-ci.org/thisisatest/thisisatest)
+[![Windows Build status](https://ci.appveyor.com/api/projects/status/github/thisisatest/thisisatest?svg=true)](https://ci.appveyor.com/project/thisisatest/thisisatest)
```

Comment:
Can you simplify to a single badge?

Reply:
done. so now it's a smaller broken PR :-)